plutus-playground-client/default.nix

```diff
@@ -2,78 +2,91 @@
 let
```

Comment:
All changes to this file are similar to those in [plutus-pab-executables/demo/pab-nami/client/default.nix](https://github.com/input-output-hk/plutus-apps/pull/559/files#diff-30bebca814a8447db36df655ce65d5ed2bc26426ec9c6be5182d1b1e10d39f2c)